configuration/pih/htmlforms/vitals.xml

```diff
@@ -461,6 +461,35 @@
 
                 </fieldset>
             </includeIf>
+
+            <!-- only allow location if an active visit or user has no retro privileges -->
+            <includeIf velocityTest="$visit.open or ($user.hasPrivilege('Task: emr.retroConsultNote') and $user.hasPrivilege('Task: emr.retroConsultNoteThisProviderOnly'))">
+                <fieldset field-separator=", ">
+
+                    <legend><uimessage code="pihcore.details"/></legend>
+                    <h3><uimessage code="pihcore.details"/></h3>
+
+                    <div style="display:none">
+                        <encounterProviderAndRole default="currentUser" encounterRole="98bf2792-3f0a-4388-81bb-c78b29c0df92"
+                                                      required="true"/>
+                    </div>
+
+                    <label>
+                        <uimessage code="emr.locationRequired"/>
+                    </label>
+                    <span>
+                        <p class="required">
+                            <field>
+                                <encounterLocation default="SessionAttribute:emrContext.sessionLocationId"
+                                                   tags="Vitals Location"/>
+                            </field>
+                        </p>
+                    </span>
+                    <div  style="display:none">
+                        <encounterDate id="encounterDate" default="now"/>
+                    </div>
+                </fieldset>
+            </includeIf>
```

Comment:
This will add another question to the "Vitals" note for all users, asking for the location.  Hopefully we are okay with this.

Reply:
@mogoodrich Will it default the location on the vitals note?  Or you need to set it?  Otherwise it seems annoying for the nurse auxilliary who just collect vitals at HUM...

Reply:
Yes, good point @lnball ... it will default to the proper location... the exception would be if the session location doesn't match the visit location (which would never happen for Mirebalais, only for systems that have multiple visit locations like Haiti HIV).